.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,7 +12,7 @@ A clear and concise description of what the bug is.
 
 **Requirements for and issue**
 - [ ] A fully functioning terraform snippet that can be copy&pasted (no outside files or ENV vars unless that's part of the issue)
-- [ ] Your version of artifactory (you can `curl` it at `$host/artifactory/api/system/version`
+- [ ] Your version of Artifactory and Xray (you can `curl` it at `$host/artifactory/api/system/version`
```

Comment:
The word `artifactory` shouldn't be here. Plus, that's actually the platform version. There is a different endpoint for xray and I don't remember it

Reply:
I think Artifactory version is important for the bug report. Adding info on how to get Xray version as well.